### ISSUE TITLE
CSS

### ISSUE THREAD

**Issue by mpmckenz** [dougenas/kwitter#18]
1. Use consistent colors:
-Light blue, blue, 'octopus' purple
2. Navbar styling
3. Components should have very similar layouts for user accessibility as discussed in person
